README.md

```diff
@@ -75,251 +60,76 @@ fn draw(frame: &mut Frame) {
-used to draw frames and interact with the terminal state. (The [`DefaultTerminal`] instance is a
-type alias for a terminal with the [`crossterm`] backend.) The [`restore`] function restores the
-terminal to its original state.
+You can also watch the [EuroRust 2024 talk] to learn about common concepts in Ratatui and what's possible to build with it.
 
-```rust
-fn main() -> std::io::Result<()> {
-    let mut terminal = ratatui::init();
-    let result = run(&mut terminal);
-    ratatui::restore();
-    result
-}
-```
-
-See the [`backend` module] and the [Backends] section of the [Ratatui Website] for more info on
-the alternate screen and raw mode.
-
-### Drawing the UI
-
-Drawing the UI is done by calling the [`Terminal::draw`] method on the terminal instance. This
-method takes a closure that is called with a [`Frame`] instance. The [`Frame`] provides the size
-of the area to draw to and allows the app to render any [`Widget`] using the provided
-[`render_widget`] method. After this closure returns, a diff is performed and only the changes
-are drawn to the terminal. See the [Widgets] section of the [Ratatui Website] for more info.
-
-The closure passed to the [`Terminal::draw`] method should handle the rendering of a full frame.
+## Built with Ratatui
 
-```rust
-use ratatui::{widgets::Paragraph, Frame};
+[![Awesome](https://awesome.re/badge-flat2.svg)][awesome-ratatui]
 
-fn run(terminal: &mut ratatui::DefaultTerminal) -> std::io::Result<()> {
-    loop {
-        terminal.draw(|frame| draw(frame))?;
-        if handle_events()? {
-            break Ok(());
-        }
-    }
-}
+Check out the [showcase] section of the website, or the [awesome-ratatui] repository
+for a curated list of awesome apps/libraries built with Ratatui!
 
-fn draw(frame: &mut Frame) {
-    let text = Paragraph::new("Hello World!");
-    frame.render_widget(text, frame.area());
-}
-```
+## Templates
 
-### Handling events
+If you're looking to get started quickly, you can use one of the available templates from the [templates] repository.
 
-Ratatui does not include any input handling. Instead event handling can be implemented by
-calling backend library methods directly. See the [Handling Events] section of the [Ratatui
-Website] for more info. For example, if you are using [Crossterm], you can use the
-[`crossterm::event`] module to handle events.
+## Alternatives
 
-```rust
-use crossterm::event::{self, Event, KeyCode, KeyEvent, KeyEventKind};
-
-fn handle_events() -> std::io::Result<bool> {
-    match event::read()? {
-        Event::Key(key) if key.kind == KeyEventKind::Press => match key.code {
-            KeyCode::Char('q') => return Ok(true),
-            // handle other key events
-            _ => {}
-        },
-        // handle other events
-        _ => {}
-    }
-    Ok(false)
-}
-```
+You might want to checkout [Cursive](https://github.com/gyscos/Cursive) or
```

Comment:
Perhaps a list and a small amount of info on each? (No more than a small sentence)

Reply:
done